modules/checklist/data/controller.php

```diff
@@ -15,11 +16,20 @@ public function get_name() {
 
 	public function register_endpoints() {
 		$this->register_endpoint( new Steps( $this ) );
+		$this->register_endpoint( new User_Progress( $this ) );
 	}
 
 	// Bypass, currently not required.
 	protected function register_index_endpoint() {}
 
+	public function create_items_permissions_check( $request ) {
+		return true;
+	}
+
+	public function create_item_permissions_check( $request ) {
+		return true;
+	}
```

Comment:
Can you add the correct permissions here?

Reply:
done